xmrstak/misc/executor.cpp

```diff
@@ -571,6 +571,13 @@ void executor::ex_main()
 			pools.emplace_front(0, "donate.xmr-stak.net:4444", "", "", "", 0.0, true, false, "", true);
 		break;
 
+	case cryptonight_aeon:
```

Comment:
This isn't needed. In cases like this it is much better to do:

`case cryptonight_lite:`
`case cryptonight_aeon:`
`    [...]`
`    break;`

Can you correct the all switch statements in your to this format?